tiatoolbox/models/engine/patch_predictor.py

```diff
@@ -117,63 +110,141 @@ class PatchPredictor(EngineABC):
          - 0.867
 
     Args:
-        model (nn.Module):
-            Use externally defined PyTorch model for prediction with.
-            weights already loaded. Default is `None`. If provided,
-            `pretrained_model` argument is ignored.
-        pretrained_model (str):
-            Name of the existing models support by tiatoolbox for
-            processing the data. For a full list of pretrained models,
-            refer to the `docs
+        model (str | ModelABC):
+            A PyTorch model. Default is `None`.
```

Comment:
would be better to update the docstring here to clarify that the user can pass a string to select from a model Zoo or provide a Pytorch model.

Reply:
Fixed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/783/commits/45a339b53a2cecd8bc7c7c722024fa2e2fb0e6ed